contracts/azorius/BaseTokenVoting.sol

```diff
@@ -17,7 +15,7 @@ abstract contract BaseTokenVoting is BaseStrategy {
         uint256 noVotes; // The total number of NO votes for this proposal
         uint256 yesVotes; // The total number of YES votes for this proposal
         uint256 abstainVotes; // The total number of ABSTAIN votes for this proposal
-        uint256 deadline; // The timestamp voting ends for this proposal
+        uint256 votingDeadline; // The timestamp voting ends for this proposal
```

Comment:
I don't believe so, pretty much everything in the contracts depends on timestamps and seconds

Reply:
Based on our discussion at standup today, sounds like we will probably move to using block numbers everywhere instead of timestamps, but will make that change in the next PR.

Reply:
yeah, sounds good